RichEditorView/Classes/RichEditorView.swift

```diff
@@ -287,23 +347,23 @@ import UIKit
     public func header(_ h: Int) {
         runJS("RE.setHeading('\(h)');")
     }
-
+    
     public func indent() {
         runJS("RE.setIndent();")
     }
-
+    
     public func outdent() {
         runJS("RE.setOutdent();")
     }
-
+    
     public func orderedList() {
         runJS("RE.setOrderedList();")
     }
-
+    
```

Comment:
It would be good to minimize formatting changes to not create too much gap with the 3rd party rep.